Tools/clinic/clinic.py

```diff
@@ -844,7 +844,9 @@ def render(
         for o in signatures:
             if isinstance(o, Function):
                 if function:
-                    fail("You may specify at most one function per block.\nFound a block containing at least two:\n\t" + repr(function) + " and " + repr(o))
+                    fail("You may specify at most one function per block. "
+                         "Found a block containing at least two: "
+                         "{function!r} and {o!r}")
```

Comment:
F-string without an `f` (which implies that this branch doesn't have coverage ;)

```suggestion
                    fail(f"You may specify at most one function per block. "
                         f"Found a block containing at least two: "
                         f"{function!r} and {o!r}")
```

Reply:
Oops!

Reply:
Is this even possible to reach using normal clinic code? Anything post the function declaration is parsed as a docstring. I don't see how it would be possible to generate such input. The only scenario I can think of is if you plug into the clinic classes (like FakeClinic does) and deliberately manipulate the blocks.

It looks to me this branch can be made into an assert.

Reply:
I can revert this error message fixup for now.

Reply:
Yes, let's leave this as it is for now and investigate more in a future issue/PR